Assets/Scripts/Enemy/ScribbleWall.cs

```diff
@@ -2,17 +2,53 @@
 using System.Collections;
```

Comment:
for the most part, the scribble wall is just an enemy that has respawn logic. can we just add the respawn logic as a 2nd script that we add to the scribble wall prefab/gameobject? or modify the code to inherit from the original enemy class?